frontend/templates/views/partials/example-sidebar.j2

```diff
@@ -29,11 +29,13 @@
                 </div>
                 <ul class="nav-list level-2">
                   {% for example in examples %}
+                  {% if not example.draft or doc.pod.env.dev %}
```

Comment:
for the future: it'd be better to have a method "publishedExampes" instead of duplicating the logic.